src/app/features/progress-dashboard-widget/edit-progress-dashboard/edit-progress-dashboard.component.html

```diff
@@ -92,8 +100,8 @@ <h1 i18n="Edit the progress of one or multiple tasks">Edit Progress</h1>
 <mat-dialog-actions>
     <button
       mat-stroked-button
-      [mat-dialog-close]="forms.value"
-      [disabled]="!forms.valid"
+      [mat-dialog-close]="outputData.value"
+      [disabled]="!parts.valid"
     >
       <span [matTooltip]="tooltipOnSave" i18n>Save</span>
```

Comment:
This getter is not really needed
```suggestion
      <span matTooltip="Fix the errors to save the form" [matTooltipDisabled]="form.valid" i18n-matTooltip="Shown when there are errors that prevent saving" i18n>Save</span>
```

Reply:
Done